frontend/src/index.tsx

```diff
@@ -3,12 +3,21 @@ import { createRoot } from "react-dom/client";
 import "./index.css";
 import * as serviceWorkerRegistration from "./serviceWorkerRegistration";
 import App from "App";
-
+import { Auth0Provider } from "@auth0/auth0-react";
+const AUTH0_CLIENT_ID = process.env.REACT_APP_AUTH0_CLIENT_ID || "clientid";
+const AUTH0_DOMAIN = process.env.REACT_APP_AUTH0_DOMAIN || "domain";
 const root = createRoot(document.getElementById("root") as HTMLElement);
-
 root.render(
   <StrictMode>
-    <App />
+    <Auth0Provider
```

Comment:
Let's move this into `App.tsx` (because we use load all the other providers there).

Reply:
done